tests/Feature/PhotosTest.php

```diff
@@ -203,4 +203,46 @@ public function test_upload_2()
 		Configs::set('SL_enable', $init_config_value1);
 		Configs::set('SL_for_admin', $init_config_value2);
 	}
+
+	public function test_upload_symlink()
+	{
+		$photos_tests = new PhotosUnitTest();
+
+		// save initial value
+		$init_config_value1 = Configs::get_value('import_via_symlink');
+
+		// enable import via symlink option
+		Configs::set('import_via_symlink', '1');
+		$this->assertEquals(Configs::get_value('import_via_symlink'), '1');
+
+		// upload the photo
+		copy('tests/Feature/night.jpg', 'public/uploads/import/night.jpg');
+		$file = new UploadedFile('public/uploads/import/night.jpg', 'night.jpg',
+			'image/jpg', null, true);
+		$id = $photos_tests->upload($this, $file);
+		$photos_tests->get($this, $id, 'true');
+
+		// check if the file is still there (without symlinks the photo would have been deleted)
+		$this->assertEquals(true, file_exists('public/uploads/import/night.jpg'));
+
+		// TODO...
+
+		// getting the photo after deleting it must fail if the symlink worked
+		$this->assertEquals(true, unlink('public/uploads/import/night.jpg'));
+		$photos_tests->get($this, $id, 'false');
+
+		// cleanup
+		$photos_tests->delete($this, $id, 'true');
```

Comment:
Seems this one is not returning 'true' as expected.

I guess the reason is that the delete part is trying to delete a file that is not here anymore.

Reply:
Yes, currently working on it. I gave up trying to run the tests locally. So I'll need to wait for Travis every time now to debug... :-P